src/plugins/vis_type_vislib/public/vislib/visualizations/gauges/meter.js

```diff
@@ -68,12 +70,19 @@ export class MeterGauge {
 
   getLabels() {
     const isPercentageMode = this.gaugeConfig.percentageMode;
+    const percentageFormatPattern =
+      this.gaugeConfig.percentageFormatPattern ||
+      this.gaugeChart.events.uiSettings.get(UI_SETTINGS.FORMAT_PERCENT_DEFAULT_PATTERN);
```

Comment:
not sure that it's a legal way for getting a uiSettings client

Reply:
Hm, What about getting uiSettings from this.gaugeChart.handler ?

Reply:
pass it from vislib/visualizations/gauge_chart.js
![image](https://user-images.githubusercontent.com/20072247/108196182-d967fa00-7129-11eb-84b1-e625cb815581.png)

Reply:
done